### PR TITLE
Flask API for relationship extraction

### PR DESCRIPTION
**Pull request by meghanabhange**

### CODE REVIEW

File: relationship_extractor.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+"Family of P13"
+"Son of P12 and Daughter of P12"
+"Friend of P12, P12, 14 and son of P13"
+```
+
+What doesn't work : Relationship without word `of`
+```
+"Friend: P12"
+"Friends with P12"
+"Wife of p12"
```
Comment (NirantK) [inline relationship_extractor.md]: This example has "of" ?

File: relationship_extractor.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Indian travelling with Italian tourist
+# Will return Indian and Italian
+```
+
+## API Details 
+API URL : http://coronatravelhistory.pythonanywhere.com/
+
+Example request :
+```
+curl --header "Content-Type: application/json" --request POST --data '{"patients":[{"patientId":"1","notes":"Student from Wuhan, recovery confirmed on 14 Feb"},{"patientId":"2","notes":"Family members of P4"},{"patientId":"3","notes":"Travel from Italy on 29/02/2020 through Doha"}]}' http://coronatravelhistory.pythonanywhere.com/
```
Comment (NirantK) [inline relationship_extractor.md:44]: Add a Python example as well? Use Postman and share that please?

**Review by NirantK — CHANGES_REQUESTED** [review]

File: relationship_extractor.md (hunk 2 above)
Comment (meghanabhange) [reply relationship_extractor.md:44]: 96a45f5ed0c6d759e88ac9821c86bf407fd2b884 solves this

File: relationship_extractor.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+```
+
+Python Example Request : 
+
+```
```
Comment (NirantK) [inline relationship_extractor.md]:
```suggestion
```python
```

**Pull request merged**